deeptrack/features.py

```diff
@@ -6004,14 +5841,14 @@ def __init__(
         key: Any
             The key used to identify the stored output.
         replace: bool, optional
-            If `True`, replaces the stored value with a new computation. 
+            If `True`, replaces the stored value with a new computation.
             Defaults to `False`.
         **kwargs:: dict of str to Any
             Additional keyword arguments passed to the parent `Feature` class.
 
         """
 
-        super().__init__(feature=feature, key=key, replace=replace, **kwargs)
+        super().__init__(key=key, replace=replace, **kwargs)
```

Comment:
The removal of feature=feature is the only "functional" change? or are there other changes?